lib/trino-hdfs/src/main/java/io/trino/hdfs/gcs/HiveGcsConfig.java

```diff
@@ -17,11 +17,43 @@
 import io.airlift.configuration.ConfigDescription;
 import io.airlift.configuration.validation.FileExists;
 
+import javax.annotation.Nullable;
+
+import static com.google.common.base.Preconditions.checkState;
+
 public class HiveGcsConfig
 {
     private boolean useGcsAccessToken;
+    private String jsonKey;
     private String jsonKeyFilePath;
 
+    public boolean isUseGcsAccessToken()
+    {
+        return useGcsAccessToken;
+    }
+
+    @Config("hive.gcs.use-access-token")
```

Comment:
Can we update `hive-gcs-tutorial.rst`? Follow-up is fine though.

Reply:
let's follow-up. Docs would need to have good advice on where this can be used, which is tricky.